src/Type/Object/PostObject.php

```diff
@@ -263,6 +263,18 @@ public static function get_post_object_fields( $post_type_object, $type_registry
 			];
 		}
 
+		if ('page' === $post_type_object->name)
+		{
+			$fields['isPostsPage'] = [
+				'type'        => ['non_null' => 'Bool'],
+				'description' => __('Whether this page is set to the blog posts page.', 'wp-graphql'),
+				'resolve'     => function (Post $page)
```

Comment:
Since the Model layer has an `isPostsPage` field, we don't need to add a resolver here. The field from the model will be used.